test/utils/testHelpers.ts

```
@@ -82,7 +120,285 @@ export class TestDataFactory {
+  }> {
+    let database: ProjectDatabase;
+    let testData: { projects: Project[]; tasks: Task[]; memories: Memory[] };
+
+    switch (scenario) {
+      case 'empty':
+        database = TestDataFactory.createDatabase();
+        testData = { projects: [], tasks: [], memories: [] };
+        break;
+
+      case 'basic':
+        const project = TestDataFactory.createProject({ name: 'Test Project' });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)